### ISSUE TITLE
Query

### ISSUE THREAD

**Issue by Hamza004** [himahuja/StreamMiner#1]
Hello,
Does "StreamMiner" do a factcheck for a given test data or is it just for checking the performance of (predpath, rel-predpath and knowledgestream) based on training data provided in data.zip.
Thanks :)

**Comment by himahuja** [comment]
No, we haven't yet worked on a separate module to fact-check test data. Since you can only query on object elements existent in the training data, you can append the test element you want to test in the corresponding test data.
